src/plugins/Dropaccelerator.cpp

```diff
@@ -7,6 +7,9 @@
 
 #include "Dropaccelerator.h"
 
+#include "Simulation.h"
```

Comment:
is this new include on purpose?

Reply:
I had to include a lot of new `Simulation.h` because of all the `global_simulation` or `_simulation`, see last paragraph in Description above.

Reply:
Ah sorry didn't realize this! Makes sense and my other comment thus is also irrelevant.

> A lot of files use global_simulation or _simulation, e.g. to get the ensemble.

All hail the great singleton! :pray: Yet another anti-pattern :grimacing: